core/templates/core/redirect_url_page.html

```diff
@@ -0,0 +1,26 @@
+{% load build_absolute_uri %}
+
+<!DOCTYPE html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- We imitate a normal page meta data so normal data can be obtained for link previews -->
+    <meta property="og:title" content="{{ shortcut.title|default:"Squire" }}" />
+    <meta property="og:description"
+          content="{{ shortcut.title|default:'Squire is the new webapp developed by the UUPS-committee of ESRG Knights of the Kitchen Table.'}}" />
```

Comment:
Squire no longer new at this point, is it possible to not hardcode this anymore?

Reply:
Created new issue for this as it's litterally the current default text as extracted from base.html and thus outside the scope of this PR